### PR TITLE
SwiftFormat

### LINKED ISSUE #77: SwiftLint autocorrect or SwiftFormat or both?
In our PRs, we had several discussions regarding using SwiftLint autocorrect or SwiftFormat. Let's discuss here what's best for us to implement. Should we use both? Or just one of them?

@kevinrenskers @amirdew your experience will be very useful here!

### PR DESCRIPTION
**Pull request by kevinrenskers**
Hoo boy, hold on to your hats! A big PR with lots of changes. It's all just formatting though. Closes #77.

(edit: well actually this one isn't that large haha)

### CODE REVIEW

**Comment by kevinrenskers** [comment]
This repo holds the actual SwiftLint config, so we can easily reuse this in Collect.

File: Danger-Swift/Sources/WeTransferPRLinter/WeTransferPRLinter.swift [hunk 1]
```diff
@@ -1,17 +1,16 @@
-// danger:disable unowned_self
```
Comment (kevinrenskers) [inline Danger-Swift/Sources/WeTransferPRLinter/WeTransferPRLinter.swift:1]: Things like this we need to move to after the imports, so it doesn't get removed.

**Comment by wetransferplatform** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Messages</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:book:</td>
      <td>

  View more details on <a href="https://app.bitrise.io/build/87e0cea3-05f6-4f39-8bec-dcc6e3e490bc" target="_blank">Bitrise</a>
  </td>
    </tr>
  
<tr>
      <td>:book:</td>
      <td>WeTransferPRLinter-Package: Executed 26 tests, with 0 failures (0 unexpected) in 1.904 (1.933) seconds</td>
    </tr>
  </tbody>
</table>

## WeTransferPRLinter.framework: Coverage: 82.03
| File | Coverage ||
| --- | --- | --- |
SwiftLintExecutor.swift | 0.0% | ⚠️
CoverageReporter.swift | 63.64% | ✅
XcodeSummaryReporter.swift | 37.04% | ✅
WeTransferPRLinter.swift | 98.92% | ✅


## WeTransferPRLinterTests.xctest: Coverage: 100.0
| File | Coverage ||
| --- | --- | --- |
DangerTestExtensions.swift | 100.0% | ✅
Mocks.swift | 100.0% | ✅
WeTransferLinterTests.swift | 100.0% | ✅

<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/swift">Danger Swift</a> against c441d984e7e03ae29438b0b95b71bd6205f45b03
</p>

**Review by AvdLee — APPROVED** [review]
I LOVE this, we should've done this so much earlier. I'm going to spend some time looking into rules which can help us improve even more. This is a good start though!

**Pull request merged**